code/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -91,10 +91,10 @@ public boolean isSpinning() {
     }
 
     public void periodic() {
-        SmartDashboard.putNumber("Arm Encoder", armEncoder.get());
+        // SmartDashboard.putNumber("Arm Encoder", armEncoder.get());
```

Comment:
Generally speaking there's no need to check in commented code, if we need it back later we can pull it out of the history